File: Basic Programs -1/Only even prime digit/prime digit.java
import java.util.Scanner;

public class EvenPrimeChecker {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String number = sc.next();
        if(number.contains("2"))
            System.out.println("Contains Even Prime");
        else
            System.out.println("Does Not Contain Even Prime");
    }
}

File: Basic Programs -1/Only even prime digit/prime digit.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string number;
    cin >> number;
    if(number.find('2') != string::npos)
        cout << "Contains Even Prime";
    else
        cout << "Does Not Contain Even Prime";
    return 0;
}
